tests/test_cli.py

```diff
@@ -305,6 +305,9 @@ def test_run_with_user_global_config_file(self):
         config = os.path.join(dir, 'config')
 
         self.addCleanup(os.environ.update, HOME=os.environ['HOME'])
+        # remove other env vars to make sure we are using the HOME config file.
+        os.environ.pop('YAMLLINT_CONFIG_FILE', None)
+        os.environ.pop('XDG_CONFIG_HOME', None)
```

Comment:
I've also patched the `test_run_with_user_global_config_file` test by adding removing other env vars.